services/settings/README.md

```diff
@@ -83,3 +83,365 @@ The `OCIS_DEFAULT_LANGUAGE` setting impacts the `notification` and `userlog` ser
 *   If  `OCIS_DEFAULT_LANGUAGE` **is set**, the expected behavior is:
     *   The `notification` and `userlog` services and the WebUI use `OCIS_DEFAULT_LANGUAGE`  by default until a user sets another language in the WebUI via _Account -> Language_.
     *   If a user sets another language in the WebUI in _Account -> Language_, the `notification` and `userlog` services and WebUI use the language defined by the user. If no translation is found, it falls back to `OCIS_DEFAULT_LANGUAGE` and then to English.
+
+## Custom Roles
+
+It is possible to replace the default ocis roles (`admin`, `user`) with custom roles that contain custom permissions. One can set `SETTINGS_BUNDLES_PATH` to the path of a `json` file containing the new roles.
```

Comment:
"It is possible to replace the default ocis roles"

I'm not sure if we want to replace default roles, but I don't see an example of how to do that. I'd rather remove that information (no need to change anything in the code), and just document how new roles can be added.
If the replacement can happen due to "an accident" if a new role overwrites a default role, that should be noted, likely as a warning

Reply:
> and just document how new roles can be added.

This is not possible in ocis as of now. The only thing you can do is to replace all existing roles with custom ones.

Reply:
Now I'm more confused. In the example you've shown how to create the "User with all available permissions" role, so it should be possible to create a "logo writer" role with just permission to create the logo (for example).

If it's something to do in the near future, it might be better to wait until it's ready to be used. Releasing the information without being ready will be confusing for the admins.

I don't think replacing the current roles is useful. In fact it could be dangerous because the user role could have all permissions while the admin role just a few, kind of reversing the roles. This is why I'd rather not document that part.
If the idea of replacing the roles is mostly to change the displayname of the role (change "admin" role for "ocis boss"), the docs should be oriented to this use case. Basically, "how to change the displayname of a role" instead of "how to add custom roles".

Reply:
>  In the example you've shown how to create the "User with all available permissions" role, so it should be possible to create a "logo writer" role with just permission to create the logo (for example)

Yes that is possible. But it is NOT possible to add new additional roles to the existing ones. This is also not planned in the foreseeable future. The feature is to REPLACE the default roles with custom ones.